.travis.yml

```diff
@@ -8,6 +8,21 @@ jobs:
     - docker build --target test  --tag todo-app:test .
     - docker run -e SECRET_KEY -e API_KEY -e API_TOKEN todo-app:test tests_e2e
     - docker run --env-file ./.env.test todo-app:test tests
+
+  - stage: docker build and publish
+    script:
+    - docker build --target production --tag kiranuppi84/todo-app:prod .
+    - echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
+    - docker push kiranuppi84/todo-app:prod
```

Comment:
Note that `kiranuppi84` is your Docker username, so we could reuse the environment variable here if we wanted to make this a little easier to vary for a new user